roles/ceph-handler/tasks/main.yml

```diff
@@ -36,4 +36,8 @@
 - name: set_fact handler_mgr_status
   set_fact:
     handler_mgr_status: "{{ (mgr_socket_stat.get('rc') == 0) if not containerized_deployment | bool else (ceph_mgr_container_stat.get('rc') == 0 and ceph_mgr_container_stat.get('stdout_lines', []) | length != 0) }}"
-  when: inventory_hostname in groups.get(mgr_group_name, [])
\ No newline at end of file
+  when: inventory_hostname in groups.get(mgr_group_name, [])
+
+- name: set_fact handler_crash_status
+  set_fact:
+    handler_crash_status: "{{ crash_process.get('rc') == 0 if not containerized_deployment | bool else (ceph_crash_container_stat.get('rc') == 0 and ceph_crash_container_stat.get('stdout_lines', []) | length != 0) }}"
```

Comment:
do we need to add a when on the core ceph groups ? (ie: mons, mgrs, osds, etc...)